tests/python/gaia-ui-tests/gaiatest/apps/gallery/app.py

```diff
@@ -73,12 +73,15 @@ def empty_gallery_text(self):
 
     @property
     def are_gallery_items_displayed(self):
-        return self.marionette.find_element(*self._gallery_items_locator).is_displayed()
+        return self.is_element_displayed(*self._gallery_items_locator)
 
     @property
     def thumbnails(self):
-        return [self.Thumbnail(self.marionette, thumbnail, index)
```

Comment:
That list comprehension looks identical to the changes you made. What are the difference?